
float32 acceptance_radius		# the optimal distance to a waypoint to switch to the next

float32 yaw_acceptance			# NaN if not set

float32 altitude_acceptance_radius # the optimal vertical distance to a waypoint to switch to the next

bool abort_landing				# true if landing should be aborted
